README.md

```diff
@@ -0,0 +1,9 @@
+# AeroGear Proposals
+This repository contains proposals, examples, PoCs for potential new mobile features to be build by the mobile team.
+
+## How to submit a proposal
+* Fork this repository 
+* Create a new directory with your proposal
+* Include a README.md which is either your proposal or the index of the proposal
+* Send a PR to master on github.com/aerogear/proposals
+* Email aerogear-dev so we know to take a look
```

Comment:
From Craig's email, I suggest we also add some guidelines here about when a proposal is required and what's the basic structure should be like.

* When a proposal is necessary

> In my experience they are often used for larger changes, breaking changes or changes in technical direction that have cross cutting concerns.

* A proposal should include:

** What
** Why
** How

WDYT?

Reply:
ON the one hand, yay structure.  On the other hand if we can't look at a proposal in a PR and answer those questions then we ask them in the PR and the proposal gets updated.

Reply:
I think outlining that this what we expect from a proposal can save time and effort on our part. It is similar to saying we expect unit tests for your pull requests. It sets a high level standard that should be adhered too before the PR is made

Reply:
Updated